app/helpers/application_form_builder.rb

```diff
@@ -36,7 +36,8 @@ def money_field(method, options = {})
     default_options = {
       class: "form-field__input",
       value: money&.amount,
-      placeholder: Money.new(0, money&.currency || Money.default_currency).format
+      placeholder: Money.new(0, money&.currency || Money.default_currency).format,
+      step: "0.01" # Not all currencies have 2 decimal places
```

Comment:
RE dynamically setting based on currency:

That's a good point and I'm wondering if we add an instance method in `Money::Currency`:

```
def step
    return 1 if @minor_unit_conversion.nil? || @minor_unit_conversion == 1
    1.0 / @minor_unit_conversion.to_f
end
```

And then use it here:

```
step: (money&.currency || Money.default_currency).step
```

Reply:
I'm just curious how that will behave when you change the dropdown value? If the default currency is EUR or USD it will be set to `0.01`, but if you change that dropdown value to JPY, it won't auto-update the money field, right? Do we need to add in some JS to handle that on change?

Reply:
Good point, wasn't thinking about that.  I think your solution works for the time being, so I'll merge this and I just opened up #625 to capture this additional requirement.

Reply:
Sweet, cheers. Curious to learn what'd be the modern Rails way to do this.

Reply:
Same, adding a JS controller just for this seems overkill, but I can't think of any other solution myself